lib/VoxeetSDK.ts

```diff
@@ -1,78 +1,14 @@
-import UserInfo from "./UserInfo";
 import VoxeetMedia from "./VoxeetMedia";
-import { Configuration } from "./Configurations";
-/*
```

Comment:
Same here regarding the header, I think it should be still there (?) - I'm not a big fan since it's redundant information but just want to be sure